File: v2/tests/fixtures/analyze_fixtures/sample.go
package main

import "fmt"

type Calculator struct {
	Value int
}

func (c *Calculator) Add(a, b int) int {
	return a + b
}

func Greet(name string) string {
	return "Hello, " + name
}

func main() {
	c := Calculator{Value: 0}
	fmt.Println(c.Add(1, 2))
	fmt.Println(Greet("World"))
}

File: examples/sample.swift
import Foundation

public protocol Greeter {
    func greet(name: String) -> String
}

public struct Greeting: Greeter {
    public let prefix: String
    private var count: Int = 0

    public init(prefix: String) {
        self.prefix = prefix
    }

    public mutating func greet(name: String) -> String {
        count += 1
        return "\(prefix), \(name)"
    }
}

final class GreetingService {
    private var greeting = Greeting(prefix: "Hello")

    func welcome(_ user: String) -> String {
        return greeting.greet(name: user)
    }
}
